infra.bs

```diff
@@ -8,13 +8,14 @@ Translation: ja https://triple-underscore.github.io/infra-ja.html
 </pre>
 
 <pre class="anchors">
-urlPrefix: https://tc39.github.io/ecma262/; spec: ECMA-262;
+urlPrefix: https://tc39.github.io/ecma262/#; spec: ECMA-262;
```

Comment:
Wow, this is bad.

Reply:
Bikeshed appears to do some sort of fixup; the links in the live document all work. I was just bothered by the inconsistency.